teamcity: failed tests on master: acceptance/TestGossipRestart

The following tests appear to have failed:

[#802247](https://teamcity.cockroachdb.com/viewLog.html?buildId=802247):


```
--- FAIL: acceptance/TestGossipRestart: TestGossipRestartFirstNodeNeedsIncoming (185.020s)
test_log_scope.go:81: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestGossipRestartFirstNodeNeedsIncoming005226955
test_log_scope.go:62: use -show-logs to present logs inline
--- FAIL: acceptance/TestGossipRestart: TestGossipRestartFirstNodeNeedsIncoming: TestGossipRestartFirstNodeNeedsIncoming/runMode=local (185.020s)
gossip_peerings_test.go:155: condition failed to evaluate within 2m0s: failed to get gossip status from node 0: Get /_status/gossip/local: unsupported protocol scheme ""
------- Stdout: -------
CockroachDB node starting at 2018-07-28 19:17:49.791256659 +0000 UTC (took 0.6s)
build:               CCL v2.1.0-alpha.20180702-935-g96296a8 @ 2018/07/28 19:07:18 (go1.10.3)
admin:               http://127.0.0.1:26258
sql:                 postgresql://root@127.0.0.1:26257?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestartFirstNodeNeedsIncoming/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/1/cockroach-temp642348525
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/1
status:              initialized new cluster
clusterID:           2c00ee7f-8be3-45b5-a154-c3bb2702e6d5
nodeID:              1
CockroachDB node starting at 2018-07-28 19:17:49.91700411 +0000 UTC (took 0.7s)
build:               CCL v2.1.0-alpha.20180702-935-g96296a8 @ 2018/07/28 19:07:18 (go1.10.3)
admin:               http://127.0.0.1:26260
sql:                 postgresql://root@127.0.0.1:26259?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestartFirstNodeNeedsIncoming/runMode=local/2
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/2/cockroach-temp218047108
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/2/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/2
status:              initialized new node, joined pre-existing cluster
clusterID:           2c00ee7f-8be3-45b5-a154-c3bb2702e6d5
nodeID:              2
CockroachDB node starting at 2018-07-28 19:17:49.964623495 +0000 UTC (took 0.7s)
build:               CCL v2.1.0-alpha.20180702-935-g96296a8 @ 2018/07/28 19:07:18 (go1.10.3)
admin:               http://127.0.0.1:26264
sql:                 postgresql://root@127.0.0.1:26263?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestartFirstNodeNeedsIncoming/runMode=local/4
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/4/cockroach-temp135299677
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/4/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/4
status:              initialized new node, joined pre-existing cluster
clusterID:           2c00ee7f-8be3-45b5-a154-c3bb2702e6d5
nodeID:              3
CockroachDB node starting at 2018-07-28 19:17:49.986883389 +0000 UTC (took 0.7s)
build:               CCL v2.1.0-alpha.20180702-935-g96296a8 @ 2018/07/28 19:07:18 (go1.10.3)
admin:               http://127.0.0.1:26262
sql:                 postgresql://root@127.0.0.1:26261?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestartFirstNodeNeedsIncoming/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/3/cockroach-temp341190973
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/3
status:              initialized new node, joined pre-existing cluster
clusterID:           2c00ee7f-8be3-45b5-a154-c3bb2702e6d5
nodeID:              4
                            [1]   3
                 liveness-  [2]   1
    liveness-    liveness.  [3]   1
    liveness.   tsd         [4]   4
   tsd          tse         [5]   4
   tse         �            [6]   3
  �            �            [7]   4
  �            �            [8]   4
  �            �            [9]   3
  �            �            [10]  3
  �            �            [11]  4
  �            �            [12]  3
  �            �            [13]  3
  �            �            [14]  3
  �            �            [15]  3
  �            �            [16]  3
  �            �            [17]  3
  �            �            [18]  3
  �            �            [19]  3
  �            �            [20]  4
  �            �            [21]  3
  �            ��             [22]  4
                            [1]   3
                 liveness-  [2]   1
    liveness-    liveness.  [3]   3
    liveness.   tsd         [4]   3
   tsd          tse         [5]   3
   tse         �            [6]   3
  �            �            [7]   4
  �            �            [8]   3
  �            �            [9]   3
  �            �            [10]  3
  �            �            [11]  3
  �            �            [12]  3
  �            �            [13]  3
  �            �            [14]  3
  �            �            [15]  3
  �            �            [16]  3
  �            �            [17]  3
  �            �            [18]  3
  �            �            [19]  3
  �            �            [20]  3
  �            �            [21]  3
  �            ��             [22]  3
                            [1]   3
                 liveness-  [2]   3
    liveness-    liveness.  [3]   3
    liveness.   tsd         [4]   3
   tsd          tse         [5]   3
   tse         �            [6]   3
  �            �            [7]   3
  �            �            [8]   3
  �            �            [9]   3
  �            �            [10]  3
  �            �            [11]  3
  �            �            [12]  3
  �            �            [13]  3
  �            �            [14]  3
  �            �            [15]  3
  �            �            [16]  3
  �            �            [17]  3
  �            �            [18]  3
  �            �            [19]  3
  �            �            [20]  3
  �            �            [21]  3
  �            ��             [22]  3
CockroachDB node starting at 2018-07-28 19:18:10.597654515 +0000 UTC (took 3.9s)
build:               CCL v2.1.0-alpha.20180702-935-g96296a8 @ 2018/07/28 19:07:18 (go1.10.3)
admin:               http://127.0.0.1:26258
sql:                 postgresql://root@127.0.0.1:26257?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestartFirstNodeNeedsIncoming/runMode=local/1
attrs:               empty
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/1/cockroach-temp723431149
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/1
status:              restarted pre-existing node
clusterID:           2c00ee7f-8be3-45b5-a154-c3bb2702e6d5
nodeID:              1
CockroachDB node starting at 2018-07-28 19:18:47.799359729 +0000 UTC (took 9.7s)
build:               CCL v2.1.0-alpha.20180702-935-g96296a8 @ 2018/07/28 19:07:18 (go1.10.3)
admin:               http://127.0.0.1:26262
sql:                 postgresql://root@127.0.0.1:26261?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestartFirstNodeNeedsIncoming/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/3/cockroach-temp558959102
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/3
status:              restarted pre-existing node
clusterID:           2c00ee7f-8be3-45b5-a154-c3bb2702e6d5
nodeID:              4
CockroachDB node starting at 2018-07-28 19:18:47.80768035 +0000 UTC (took 9.7s)
build:               CCL v2.1.0-alpha.20180702-935-g96296a8 @ 2018/07/28 19:07:18 (go1.10.3)
admin:               http://127.0.0.1:26264
sql:                 postgresql://root@127.0.0.1:26263?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestartFirstNodeNeedsIncoming/runMode=local/4
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/4/cockroach-temp776537991
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/4/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/4
status:              restarted pre-existing node
clusterID:           2c00ee7f-8be3-45b5-a154-c3bb2702e6d5
nodeID:              3
CockroachDB node starting at 2018-07-28 19:18:47.809989657 +0000 UTC (took 9.7s)
build:               CCL v2.1.0-alpha.20180702-935-g96296a8 @ 2018/07/28 19:07:18 (go1.10.3)
admin:               http://127.0.0.1:26260
sql:                 postgresql://root@127.0.0.1:26259?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestartFirstNodeNeedsIncoming/runMode=local/2
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/2/cockroach-temp710589328
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/2/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/2
status:              restarted pre-existing node
clusterID:           2c00ee7f-8be3-45b5-a154-c3bb2702e6d5
nodeID:              2
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestGossipRestartFirstNodeNeedsIncoming005226955
--- FAIL: acceptance/TestGossipRestart: TestGossipRestartFirstNodeNeedsIncoming (185.020s)
test_log_scope.go:81: test logs captured to: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestGossipRestartFirstNodeNeedsIncoming005226955
test_log_scope.go:62: use -show-logs to present logs inline
--- FAIL: acceptance/TestGossipRestart: TestGossipRestartFirstNodeNeedsIncoming: TestGossipRestartFirstNodeNeedsIncoming/runMode=local (185.020s)
gossip_peerings_test.go:155: condition failed to evaluate within 2m0s: failed to get gossip status from node 0: Get /_status/gossip/local: unsupported protocol scheme ""
------- Stdout: -------
CockroachDB node starting at 2018-07-28 19:17:49.791256659 +0000 UTC (took 0.6s)
build:               CCL v2.1.0-alpha.20180702-935-g96296a8 @ 2018/07/28 19:07:18 (go1.10.3)
admin:               http://127.0.0.1:26258
sql:                 postgresql://root@127.0.0.1:26257?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestartFirstNodeNeedsIncoming/runMode=local/1
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/1/cockroach-temp642348525
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/1
status:              initialized new cluster
clusterID:           2c00ee7f-8be3-45b5-a154-c3bb2702e6d5
nodeID:              1
CockroachDB node starting at 2018-07-28 19:17:49.91700411 +0000 UTC (took 0.7s)
build:               CCL v2.1.0-alpha.20180702-935-g96296a8 @ 2018/07/28 19:07:18 (go1.10.3)
admin:               http://127.0.0.1:26260
sql:                 postgresql://root@127.0.0.1:26259?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestartFirstNodeNeedsIncoming/runMode=local/2
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/2/cockroach-temp218047108
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/2/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/2
status:              initialized new node, joined pre-existing cluster
clusterID:           2c00ee7f-8be3-45b5-a154-c3bb2702e6d5
nodeID:              2
CockroachDB node starting at 2018-07-28 19:17:49.964623495 +0000 UTC (took 0.7s)
build:               CCL v2.1.0-alpha.20180702-935-g96296a8 @ 2018/07/28 19:07:18 (go1.10.3)
admin:               http://127.0.0.1:26264
sql:                 postgresql://root@127.0.0.1:26263?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestartFirstNodeNeedsIncoming/runMode=local/4
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/4/cockroach-temp135299677
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/4/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/4
status:              initialized new node, joined pre-existing cluster
clusterID:           2c00ee7f-8be3-45b5-a154-c3bb2702e6d5
nodeID:              3
CockroachDB node starting at 2018-07-28 19:17:49.986883389 +0000 UTC (took 0.7s)
build:               CCL v2.1.0-alpha.20180702-935-g96296a8 @ 2018/07/28 19:07:18 (go1.10.3)
admin:               http://127.0.0.1:26262
sql:                 postgresql://root@127.0.0.1:26261?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestartFirstNodeNeedsIncoming/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/3/cockroach-temp341190973
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/3
status:              initialized new node, joined pre-existing cluster
clusterID:           2c00ee7f-8be3-45b5-a154-c3bb2702e6d5
nodeID:              4
                            [1]   3
                 liveness-  [2]   1
    liveness-    liveness.  [3]   1
    liveness.   tsd         [4]   4
   tsd          tse         [5]   4
   tse         �            [6]   3
  �            �            [7]   4
  �            �            [8]   4
  �            �            [9]   3
  �            �            [10]  3
  �            �            [11]  4
  �            �            [12]  3
  �            �            [13]  3
  �            �            [14]  3
  �            �            [15]  3
  �            �            [16]  3
  �            �            [17]  3
  �            �            [18]  3
  �            �            [19]  3
  �            �            [20]  4
  �            �            [21]  3
  �            ��             [22]  4
                            [1]   3
                 liveness-  [2]   1
    liveness-    liveness.  [3]   3
    liveness.   tsd         [4]   3
   tsd          tse         [5]   3
   tse         �            [6]   3
  �            �            [7]   4
  �            �            [8]   3
  �            �            [9]   3
  �            �            [10]  3
  �            �            [11]  3
  �            �            [12]  3
  �            �            [13]  3
  �            �            [14]  3
  �            �            [15]  3
  �            �            [16]  3
  �            �            [17]  3
  �            �            [18]  3
  �            �            [19]  3
  �            �            [20]  3
  �            �            [21]  3
  �            ��             [22]  3
                            [1]   3
                 liveness-  [2]   3
    liveness-    liveness.  [3]   3
    liveness.   tsd         [4]   3
   tsd          tse         [5]   3
   tse         �            [6]   3
  �            �            [7]   3
  �            �            [8]   3
  �            �            [9]   3
  �            �            [10]  3
  �            �            [11]  3
  �            �            [12]  3
  �            �            [13]  3
  �            �            [14]  3
  �            �            [15]  3
  �            �            [16]  3
  �            �            [17]  3
  �            �            [18]  3
  �            �            [19]  3
  �            �            [20]  3
  �            �            [21]  3
  �            ��             [22]  3
CockroachDB node starting at 2018-07-28 19:18:10.597654515 +0000 UTC (took 3.9s)
build:               CCL v2.1.0-alpha.20180702-935-g96296a8 @ 2018/07/28 19:07:18 (go1.10.3)
admin:               http://127.0.0.1:26258
sql:                 postgresql://root@127.0.0.1:26257?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestartFirstNodeNeedsIncoming/runMode=local/1
attrs:               empty
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/1/cockroach-temp723431149
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/1/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/1
status:              restarted pre-existing node
clusterID:           2c00ee7f-8be3-45b5-a154-c3bb2702e6d5
nodeID:              1
CockroachDB node starting at 2018-07-28 19:18:47.799359729 +0000 UTC (took 9.7s)
build:               CCL v2.1.0-alpha.20180702-935-g96296a8 @ 2018/07/28 19:07:18 (go1.10.3)
admin:               http://127.0.0.1:26262
sql:                 postgresql://root@127.0.0.1:26261?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestartFirstNodeNeedsIncoming/runMode=local/3
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/3/cockroach-temp558959102
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/3/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/3
status:              restarted pre-existing node
clusterID:           2c00ee7f-8be3-45b5-a154-c3bb2702e6d5
nodeID:              4
CockroachDB node starting at 2018-07-28 19:18:47.80768035 +0000 UTC (took 9.7s)
build:               CCL v2.1.0-alpha.20180702-935-g96296a8 @ 2018/07/28 19:07:18 (go1.10.3)
admin:               http://127.0.0.1:26264
sql:                 postgresql://root@127.0.0.1:26263?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestartFirstNodeNeedsIncoming/runMode=local/4
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/4/cockroach-temp776537991
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/4/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/4
status:              restarted pre-existing node
clusterID:           2c00ee7f-8be3-45b5-a154-c3bb2702e6d5
nodeID:              3
CockroachDB node starting at 2018-07-28 19:18:47.809989657 +0000 UTC (took 9.7s)
build:               CCL v2.1.0-alpha.20180702-935-g96296a8 @ 2018/07/28 19:07:18 (go1.10.3)
admin:               http://127.0.0.1:26260
sql:                 postgresql://root@127.0.0.1:26259?sslmode=disable
logs:                /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/TestGossipRestartFirstNodeNeedsIncoming/runMode=local/2
temp dir:            /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/2/cockroach-temp710589328
external I/O path:   /home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/2/extern
store[0]:            path=/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/acceptance/.localcluster015968430/2
status:              restarted pre-existing node
clusterID:           2c00ee7f-8be3-45b5-a154-c3bb2702e6d5
nodeID:              2
test logs left over in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/acceptance/logTestGossipRestartFirstNodeNeedsIncoming005226955

```

Please assign, take a look and update the issue accordingly.

--- a/pkg/acceptance/gossip_peerings_test.go
+++ b/pkg/acceptance/gossip_peerings_test.go
@@ -206,7 +206,6 @@ func testClusterConnectedAndFunctional(ctx context.Context, t *testing.T, c clus
 //
 // See https://github.com/cockroachdb/cockroach/issues/18027.
 func TestGossipRestartFirstNodeNeedsIncoming(t *testing.T) {
-	t.Skip("#28004")
 	s := log.Scope(t)
 	defer s.Close(t)
 

--- a/pkg/storage/store.go
+++ b/pkg/storage/store.go
@@ -1700,12 +1700,11 @@ func (s *Store) asyncGossipStore(ctx context.Context, reason string, useCached b
 
 // GossipStore broadcasts the store on the gossip network.
 func (s *Store) GossipStore(ctx context.Context, useCached bool) error {
-	// This should always return immediately and acts as a sanity check that we
-	// don't try to gossip before we're connected.
 	select {
 	case <-s.cfg.Gossip.Connected:
 	default:
-		log.Fatalf(ctx, "not connected to gossip")
+		// Nothing to do if gossip is not connected.
+		return nil
 	}
 
 	// Temporarily indicate that we're gossiping the store capacity to avoid

